spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -288,182 +313,121 @@ public interface PolledBarista {
-
-    private MessageChannel output;
-
-    @Autowired
-    public SendingBean(@Qualifier("customOutput") MessageChannel output) {
-        this.output = output;
-    }
-
-    public void sayHello(String name) {
-         this.output.send(MessageBuilder.withPayload(name).build());
-    }
-}
+@Autowire
+@Qualifier("myChannel")
+private MessageChannel output;
 ----
 
 [[spring-cloud-stream-overview-producing-consuming-messages]]
-==== Producing and Consuming Messages
+=== Producing and Consuming Messages
 
 You can write a Spring Cloud Stream application by using either Spring Integration annotations or Spring Cloud Stream's `@StreamListener` annotation.
-The `@StreamListener` annotation is modeled after other Spring Messaging annotations (such as `@MessageMapping`, `@JmsListener`, `@RabbitListener`, and so on) but adds content type management and type coercion features.
+The `@StreamListener` annotation is modeled after other Spring Messaging annotations (such as `@MessageMapping`, `@JmsListener`, `@RabbitListener`, and so on) but
+conviniences such as content based routing and others.
+
+==== Native Spring Integration Support
 
-===== Native Spring Integration Support
+Spring Cloud Stream is build on the concepts and patterns defined by http://www.enterpriseintegrationpatterns.com/[Enterprise Integration Patterns] and relies
+in its internal implementation on an already established and popular implementation of Enterprise Integration Patterns within Spring portfolio of projects -
+https://projects.spring.io/spring-integration/[Spring Integration] framework.
 
-Because Spring Cloud Stream is based on Spring Integration, Stream completely inherits Integration's foundation and infrastructure as well as the component itself.
-For example, you can attach the  output channel of a `Source` to a `MessageSource`, as follows:
+So it's only natiural for it to support the foundation, semantics and configuration options that are already established by Spring Integration
+
+For example, you can attach the output channel of a `Source` to a `MessageSource` and use all familiar `@InboundChannelAdapter` annotation, as follows:
 
 [source, java]
 ----
 @EnableBinding(Source.class)
 public class TimerSource {
 
-  @Value("${format}")
-  private String format;
-
   @Bean
-  @InboundChannelAdapter(value = Source.OUTPUT, poller = @Poller(fixedDelay = "${fixedDelay}", maxMessagesPerPoll = "1"))
+  @InboundChannelAdapter(value = Source.OUTPUT, poller = @Poller(fixedDelay = "10", maxMessagesPerPoll = "1"))
   public MessageSource<String> timerMessageSource() {
-    return () -> new GenericMessage<>(new SimpleDateFormat(format).format(new Date()));
+    return () -> new GenericMessage<>("Hello Spring Cloud Stream");
   }
 }
 ----
 
-Similarly, you can use a processor's channels in a transformer, as follows:
+Similarly, you can use @Transformer and/or @ServiceActivator while providing an implementation of a message handler method for `Processor` binding contract:
```

Comment:
Replace with "Similarly, you can use `@Transformer` or `@ServiceActivator` while providing an implementation of a message handler method for a `Processor` binding contract, as shown in the following example:"